### PR TITLE
Added title to location reports

### PR DESCRIPTION
**Pull request by hayleymoodley**

### CODE REVIEW

**Comment by hayleymoodley** [comment]
@Kenneth-Siu 
@Arthur91S

**Comment by Kenneth-Siu** [comment]
Can you add your migration please? :)

File: AgentDiscoveries-Backend/src/main/resources/db/migration/V7_add_location_report_title.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE location_reports
+ADD title mediumtext;
```
Comment (Kenneth-Siu) [inline AgentDiscoveries-Backend/src/main/resources/db/migration/V7_add_location_report_title.sql:2]: Did you choose mediumtext specifically? What are the advantages of using mediumtext over other types, like varchar(255)?

Comment (hayleymoodley) [reply AgentDiscoveries-Backend/src/main/resources/db/migration/V7_add_location_report_title.sql]: No, I changed the type to Varchar(255)

**Comment by Kenneth-Siu** [comment]
Looks good! Looks like there are merge conflicts now: have a look at the Resolve conflicts button and see if you can work out what's going on. It'll display the changes from:

* the branch you're merging into, and
* the branch you're merging from

Edit the conflict files until they look like what you'd expect them to look like (make sure to remove the >>>>>>> bits)

Make sure you update your local machine's git repository to match github's by doing a git pull

**Pull request closed without merging**